python/tvm/relay/frontend/onnx.py

```diff
@@ -836,6 +837,192 @@ def _impl_v1(cls, inputs, attr, params):
         return Gelu._impl_v1([inp], attr, params)
 
 
+class EmbedLayerNormalization(OnnxOpConverter):
+    @classmethod
+    def _impl_v1(cls, inputs, attr, params):
+        input_ids = inputs[0]
+        segment_ids = inputs[1]
+        word_emb = inputs[2]
+        pos_emb = inputs[3]
+        segment_emb = inputs[4]
+        gamma = inputs[5]
+        beta = inputs[6]
+
+        mask = inputs[7]
```

Comment:
Mask and position_ids are optional, I believe you need to check the length. On the docs, it says the inputs can be a list of length 2-9 (I assume segment_emb is fine even if it's optional)

Reply:
the inputs to these converters are actually wrapped in an `onnx_input` object which returns None for out of bounds accesses, so we can safely do this (we also can't use `len` properly here as trailing optional inputs aren't counted)